src/main/java/tech/aroma/data/sql/ModuleSQLRepositories.kt

```diff
@@ -29,15 +29,16 @@ internal class ModuleSQLRepositories : AbstractModule()
 {
     override fun configure()
     {
+        install(ModuleSerializers())
```

Comment:
Ok that makes sense. ![bitmoji](https://render.bitstrips.com/v2/cpanel/7195995-288439948_4-s1-v1.png?transparent=1&palette=1&width=246)